CONTRIBUTING.md

```diff
@@ -40,6 +40,16 @@ Configuration is available for cmake, [documented elsewhere](https://cmake.org/d
 
 Mac support is pretty early and while it will compile, it's not usable for debugging yet. Builds happen with cmake the same way as Linux.
 
+### Android
+
+To build the components required to debug an Android target, first gather everything from [Dependencies](#dependencies) below, then run:
+```
+mkdir build-android
+cd build-android
+cmake -DBUILD_ANDROID=On -DANDROID_ABI=armeabi-v7a -DANDROID_NATIVE_API_LEVEL=23 ..
+make
+```
```

Comment:
Okay, done.  I don't have the Windows for Android build set up, so I haven't verified the steps.  I tend to use cygwin for tracking projects and Visual Studio Command Prompt for command line compiles, including for Android.  I haven't tried getting that to work for RenderDoc - wasn't sure if anyone was actually compiling on Windows.